### DISCUSSION TITLE
"Na" on the periodic table is generally known as what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#391] [Galaxy Trivia] [answered]
**Hint:** from Latin natrium

**Comment by peterbaccaro** [comment, marked as answer]
**Sodium** is the name of the element that has a chemical symbol of **Na** on the periodic table.

**Comment by thuongtruong1009** [comment]
"Na" on the periodic table is generally known as Sodium.
